### PR TITLE
Passing secrets to reusable workflows

### LINKED ISSUE #21850: Secrets for reusable workflows
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

https://docs.github.com/en/actions/security-guides/encrypted-secrets?tool=webui#using-encrypted-secrets-in-a-workflow

### What part(s) of the article would you like to see updated?

This article mentions using encrypted secrets in GitHub Actions. However, secrets are not automatically passed to reusable workflows. There is a separate article for reusable workflows describing this. My proposal is that a note on using secrets in reusable workflows, and a link to the separate article is mentioned in the article above as well.

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by pereriksson**
### Why:

Closes #21850

### What's being changed (if available, include any code snippets, screenshots, or gifs):

This patch contains a note on using secrets in reusable workflows.

### Check off the following:

- [x] I have reviewed my changes in staging (look for the "Automatically generated comment" and click the links in the "Preview" column to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### CODE REVIEW

**Comment by cmwilson21** [comment]
@pereriksson Thanks for submitting this PR! I edited your post slightly to link it to the issue.

I'll get this triaged for review! ⚡

File: content/actions/security-guides/encrypted-secrets.md [hunk 1]
```diff
@@ -217,6 +217,12 @@ You can check which access policies are being applied to a secret in your organi
 
 {% endnote %}
 
+{% note %}
+
+**Note:** Secrets are not automatically passed to reusable workflows. For more information, see "[Passing inputs and secrets to a reusable workflow](/actions/using-workflows/reusing-workflows#passing-inputs-and-secrets-to-a-reusable-workflow).".
```
Comment (lecoursen) [inline content/actions/security-guides/encrypted-secrets.md:222]:
```suggestion
**Note:** Secrets are not automatically passed to reusable workflows. For more information, see "[Reusing workflows](/actions/using-workflows/reusing-workflows#passing-inputs-and-secrets-to-a-reusable-workflow)."
```

File: content/actions/security-guides/encrypted-secrets.md [hunk 2]
```diff
@@ -217,6 +217,12 @@ You can check which access policies are being applied to a secret in your organi
 
 {% endnote %}
 
+{% note %}
```
Comment (lecoursen) [inline content/actions/security-guides/encrypted-secrets.md:220]:
```suggestion
{% ifversion fpt or ghec or ghes > 3.3 or ghae > 3.3 %}
{% note %}
```

File: content/actions/security-guides/encrypted-secrets.md [hunk 3]
```diff
@@ -217,6 +217,12 @@ You can check which access policies are being applied to a secret in your organi
...
+
+**Note:** Secrets are not automatically passed to reusable workflows. For more information, see "[Passing inputs and secrets to a reusable workflow](/actions/using-workflows/reusing-workflows#passing-inputs-and-secrets-to-a-reusable-workflow).".
+
+{% endnote %}
```
Comment (lecoursen) [inline content/actions/security-guides/encrypted-secrets.md:224]:
```suggestion
{% endnote %}
{% endif %}
```

**Review by lecoursen — DISMISSED** [review]
Thanks @pereriksson! The link checker is failing because the article you're linking to only exists for FPT, GHEC, and GHES/GHAE 3.4+, so I'm adding some versioning around the new note to only be included in those versions of the docs. I'm also slightly tweaking the link syntax (we use article names rather than section headings in our anchor text) and then I'll check it on on preview. 😎

**Comment by lecoursen** [comment]
Actually, I just realized this will be two notes in a row. Instead, let's update the existing note to contain both, like this:

> **Notes**:
>   - EXISTING NOTE CONTENT
>  {-% ifversion fpt or ghec or ghes > 3.3 or ghae > 3.3 %}
>   - NEW NOTE CONTENT
> {%- endif %}

Feel free to request another review from me after you've done that. Thanks!

**Comment by TPVdZoHteuquuW5uZHR5Jac88vrgQnUYm7** [comment]
Sent from my Galaxy
-------- Original message --------From: Laura Coursen ***@***.***> Date: 09/11/2022  1:51 am  (GMT+06:00) To: github/docs ***@***.***> Cc: Subscribed ***@***.***> Subject: Re: [github/docs] Passing secrets to reusable workflows (PR #21903) 
Actually, I just realized this will be two notes in a row. Instead, let's update the existing note to contain both, like this:

Notes:

EXISTING NOTE CONTENT
{-% ifversion fpt or ghec or ghes > 3.3 or ghae > 3.3 %}
NEW NOTE CONTENT
{%- endif %}


Feel free to request another review from me after you've done that. Thanks!

**Comment by pereriksson** [comment]
@lecoursen pushed, is there any issue with the Azure preview env? I wanted to check the result before re-requesting a review from you :-)

**Comment by pereriksson** [comment]
Still no luck with getting a preview environment for this PR. Let's hope Github staff can fix at some point.

**Comment by lecoursen** [comment]
> Still no luck with getting a preview environment for this PR. Let's hope Github staff can fix at some point.

Sorry I didn't mention this sooner but I've been working with the team to figure out why the preview deployment seems to be broken in this PR since you first mentioned it! Hopefully we'll get an answer soon. 🤞🏻

**Comment by Ebonsignori** [comment]
Testing to see if we can close -> reopen to fix this. Hopefully [this](https://github.com/github/docs/actions/runs/3439565296) workflow that was triggered on close will allow us to rerun the preview deploy without error

**Review by cmwilson21 — APPROVED** [review]

**Pull request merged**